README.md

```diff
@@ -94,3 +94,23 @@ The `update` orb performs an update to a given Pulumi stack.
 | stack           | string  | (none)      | Name of the Pulumi stack to update. |
 | working_directory | string | . | The relative working directory to run `pulumi` from. | 
 | skip-preview | boolean | false | Whether or not to skip the preview step before the update. | 
+
+### pulumi/destroy
+
+The `destroy` orb destroys a given Pulumi stack.
```

Comment:
Given the scope of the command, we should probably make this sound a little more scary :)

Worth cribbing a few lines from the documentation:
https://www.pulumi.com/docs/reference/cli/pulumi_destroy/

"After running to completion, all of this stack’s resources and associated state will be gone."

"Warning: this command is generally irreversible and should be used with great care."